src/app/components/card/card.component.html

```diff
@@ -1,56 +1,50 @@
-<mat-card class="card" [class.public]="publicOrg" [style.borderTopColor]="card?.color" style="border-top: 3px solid;">
-    <mat-card-header class="header">
-
-        <mat-icon mat-card-avatar matTooltip="{{'Private' | translate}}" *ngIf="this.status === 'private'" >lock</mat-icon>
-        <mat-icon mat-card-avatar matTooltip="{{'Public' | translate}}" *ngIf="this.status === 'public'" >public</mat-icon>
-        <mat-icon mat-card-avatar matTooltip="{{'Shared' | translate}}" *ngIf="this.status === 'shared'" >groups</mat-icon>
-        <mat-card-title (click)="clickOnAction(action.VIEW)" matTooltip="{{'View' | translate}}&nbsp;{{card?.title}}" [matTooltipPosition]="'above'">
-            {{card?.title}}
-        </mat-card-title>
-        <mat-card-subtitle class="subtitle">{{ 'Last update' | translate }} : {{card?.last_update_date | date}}</mat-card-subtitle>
-        <arlas-config-menu #configMenu class="config-menu" matTooltip="{{'Menu' | translate}}" [zone]="'config.json'"
-            [actions]="card?.actions" (actionExecutedEmitter)="afterAction($event)"></arlas-config-menu>
-    </mat-card-header>
+<mat-card  class="card mat-elevation-z1">
+    <div (click)="clickOnAction(action.VIEW)" class="card_preview" [class.no-preview]="card?.preview === 'assets/no_preview.png'">
```

Comment:
I think we should use a variable for the no_preview asset, to avoid missing it if we change something in the app structure